app/models/validations/household_validations.rb

```diff
@@ -64,6 +64,22 @@ def validate_previous_housing_situation(record)
     if record.is_relet_to_temp_tenant? && !record.previous_tenancy_was_temporary?
       record.errors.add :prevten, I18n.t("validations.household.prevten.non_temp_accommodation")
     end
+
+    if record.age1.present? && record.age1 > 19 && record.prevten == 13
```

Comment:
Possibly more readable to add a wrapper method like `previous_tenancy_was_foster_care?` to `case_log`?